plugins/plugin-annotations/src/logo.mjs

```diff
@@ -1,8 +1,10 @@
+      `<g id="logo" transform="scale(${scale}) translate(-23 -36)"><path class="logo" stroke="none" fill="currentColor" d=${logoPath} /></g>`,
```

Comment:
Seems like you forgot to quote the path string

Reply:
If you look closely, this is using template literals - `<g> ...        ${logoPath}         ...</g>`, so it doesn't need quotes as logoPath is itself a string in `../../../sites/shared/components/logos/freesewing.mjs`

Reply:
Yes, but this will inject the string without quotes. So your code will generate this:

```svg
<path class="logo" stroke="none" fill="currentColor" d=M 35.222,0 C 34.233,0... />
```

But what we need is valid SVG output, like this:
```
<path class="logo" stroke="none" fill="currentColor" d="M 35.222,0 C 34.233,0..." />
```

So, while it is true that from a pure Javascript point of view, it is not required to put quotes here, it would result in invalid XML/SVG syntax. That's why that quotes are required.

Reply:
I have made the required changes.